target/linux/ipq40xx/files-6.6/arch/arm/boot/dts/qcom/qcom-ipq4029-ws-ap3915i.dts

```diff
@@ -10,6 +10,8 @@
 	compatible = "extreme-networks,ws-ap3915i";
 
 	aliases {
+		// TODO: Verify if the ethernet0 alias is needed
+		ethernet0 = &gmac;
```

Comment:
This is not needed for AP3915i - tested here:
https://github.com/openwrt/openwrt/commit/f7398101db7a2ddbd037658c74311fad79f9cda3
MAC-Address is fine on openwrt-24.10

Or would this make things easier..?

Quite sure the ap391x below behaves similar